v2/concordium/types.proto

```diff
@@ -2875,13 +2925,48 @@ message ChainParametersV1 {
   AuthorizationsV1 level2_keys = 14;
 }
 
+// Updatable chain parameters that apply to protocol versions 6.
+message ChainParametersV2 {
+  // Consensus parameters.
+  ConsensusParametersV1 consensus_parameters = 1;
+  // Euro per energy exchange rate.
+  ExchangeRate euro_per_energy = 2;
+  // Micro CCD per euro exchange rate.
+  ExchangeRate micro_ccd_per_euro = 3;
+  // Extra number of epochs before reduction in stake, or baker
+  // deregistration is completed.
+  CooldownParametersCpv1 cooldown_parameters = 4;
+  TimeParametersCpv1 time_parameters = 5;
```

Comment:
Missing doc

Reply:
Added some documentation for the time parameters now.